bundles/ui/org.eclipse.smarthome.ui.basic/README.md

```diff
@@ -0,0 +1,22 @@
+## Eclipse Smarthome BasicUI
+
+BasicUI is a web interface for Eclipse Smarthome based on Material Design Lite from Google.
+
+Features:
+* Responsive layout suitable for various screen sizes
+* AJAX navigation
+* Live update<sup>1</sup>
+
+<sup>1</sup>: as for now, only control states and icons are updated. Visibility changes require a page reload.
+
+Configuration:
+```
+org.eclipse.smarthome.basicui:defaultSitemap=demo
+# Icons can be disabled
+org.eclipse.smarthome.basicui:enableIcons=true
+```
+
+Screenshots:
+
+![screenshot from 2015-10-13 01 33 22](https://cloud.githubusercontent.com/assets/2057191/10440943/3cf6a2e4-714c-11e5-9580-14bbd31bf7cf.png)
```

Comment:
Since images can either be inserted in a way compatible with Github online preview (e. g. something like `../blob/master/img/image1.png?raw=true`) or linked locally (`img/image1.png`), maybe it would it be better to include README as html and not Markdown?